src/main/java/baseball/model/User.java

```diff
@@ -0,0 +1,39 @@
+package baseball.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class User {
+    private List<Integer> userNumbers;
+
+    public void inputNumber(String numberString) {
+        validateNumberDigits(numberString);
+        userNumbers = numberStringToList(numberString);
+        validateNumberUnique(userNumbers);
+    }
+
+    private List<Integer> numberStringToList(String number) {
+        List<Integer> numbers = new ArrayList<>();
+        for (String digit : number.split("")) {
+            numbers.add(Integer.parseInt(digit));
+        }
+        return numbers;
+    }
```

Comment:
스트림 API를 사용하는 방법도 있어요 :)

```suggestion
    private List<Integer> numberStringToList(String number) {
        return Arrays.stream(numbers.split(""))
          .map(Integer::parseInt)
          .toList();
    }

```

Reply:
스트림이 훨씬 깔끔하긴하네요! 다음부턴 Java API를 적극 활용해봐야겠습니다!